docs/get-started/getting-started-gcp.md

```diff
@@ -16,6 +23,102 @@ parent: Getting-Started
   GPUs](#submit-spark-jobs-to-a-dataproc-cluster-accelerated-by-gpus)
 * [Build custom Dataproc image to accelerate cluster initialization time](#build-custom-dataproc-image-to-accelerate-cluster-init-time)
 
+## Quick Start Prerequisites
+
+* gcloud CLI is installed: https://cloud.google.com/sdk/docs/install
+* python 3.8+
+* `pip install spark-rapids-user-tools`
+
+## Qualify CPU Workloads for GPU Acceleration
+
+The qualification tool is run on a Dataproc cluster that has applications that have already run.
```

Comment:
Can we rephrase this?
We can also add reference to the qualification tool.

Reply:
Fixed in latest commit